Dockerfile

```diff
@@ -29,28 +29,20 @@ COPY pytest.ini ${TPF_HOME}/pytest.ini
 COPY SAST/requirements.txt ${SAST_DIR}/requirements.txt
 COPY SAST/sast-config.yaml ${SAST_DIR}/sast-config.yaml
 
-ARG TESTS_DIR="qualitytests"
-COPY ${TESTS_DIR}/requirements.txt ${TPF_HOME}/${TESTS_DIR}/requirements.txt
-
 ARG REQUIREMENTS_FILE
 COPY ${REQUIREMENTS_FILE} ${TPF_HOME}/${REQUIREMENTS_FILE}
 RUN pip install -r ${TPF_HOME}/${REQUIREMENTS_FILE}
 
-ARG JOERN_VERSION="v1.1.1538"
+ARG JOERN_VERSION="v1.1.1709"
 RUN echo ${JOERN_VERSION}
 COPY discovery ${DISCOVERY_HOME}
 RUN chmod +x ${DISCOVERY_HOME}/joern/joern-install.sh
-RUN /bin/sh -c 'cd ${DISCOVERY_HOME}/joern/ && ./joern-install.sh --version=v1.1.1538 --install-dir=/opt/joern'
-
-# install js2cpg
-# RUN /bin/sh -c 'cd ${DISCOVERY_HOME}/joern/js2cpg/; sbt stage'
-
+RUN /bin/sh -c 'cd ${DISCOVERY_HOME}/joern/ && ./joern-install.sh --version=${JOERN_VERSION} --install-dir=/opt/joern --reinstall --without-plugins'

```

Comment:
Correct, the querydb is only really needed for the `joern-scan` utility.
It's a collection of joern scripts (similar to our pattern collection), with rules scanning for common vulnerabilities. 
It would crash the main joern component, because it gets loaded as an extension.

I'm pretty sure we don't need it, if anything runs into issues please let me know.